src/common/snippets/src/lowered/loop_info.cpp

```diff
@@ -364,24 +355,67 @@ void UnifiedLoopInfo::replace_with_new_ports(const ExpressionPort& actual_port,
     sort_ports();
 }
 
+InnerSplittedUnifiedLoopInfo::InnerSplittedUnifiedLoopInfo(size_t increment, const std::vector<LoopPort>& entries, const std::vector<LoopPort>& exits,
+                                                           const std::vector<LoopPortDesc>& in_descs, const std::vector<LoopPortDesc>& out_descs,
+                                                           const SpecificIterationHandlers& handlers, LoopInfoPtr outer_splitted_loop_info)
+    : UnifiedLoopInfo(utils::get_dynamic_value<size_t>(), increment, entries, exits, in_descs, out_descs, handlers),
+      m_outer_splitted_loop_info(std::move(outer_splitted_loop_info)) {
+    OPENVINO_ASSERT(m_outer_splitted_loop_info != nullptr, "Outer Splitted Loop Info is missed!");
+}
+
+std::shared_ptr<LoopInfo> InnerSplittedUnifiedLoopInfo::clone_with_new_expr(const ExpressionMap& expr_map, LoopInfoMap& loop_map) const {
+    if (loop_map.count(m_outer_splitted_loop_info.get()) == 0)
+        loop_map[m_outer_splitted_loop_info.get()] = m_outer_splitted_loop_info->clone_with_new_expr(expr_map, loop_map);
+
+    const auto cloned_outer_splitted_loop_info = loop_map.at(m_outer_splitted_loop_info.get());
+    const auto& new_input_ports = clone_loop_ports(expr_map, m_input_ports);
+    const auto& new_output_ports = clone_loop_ports(expr_map, m_output_ports);
+
+    return std::make_shared<InnerSplittedUnifiedLoopInfo>(m_increment, new_input_ports, new_output_ports,
+                                                          m_input_port_descs, m_output_port_descs, m_handlers,
+                                                          std::move(cloned_outer_splitted_loop_info));
+}
```

Comment:
Minor: we can't std::move const variable, just make `cloned_outer_splitted_loop_info` non const

Same for L418

Reply:
Oh, right, thanks!
Done